### PR TITLE
Refactor Person to Client

### PR DESCRIPTION
**Pull request by anunayajoshi**
I have refactored Person to Client in the main and test files. Please do help to keep a look out if I have missed any.

### CODE REVIEW

File: src/main/java/seedu/address/model/client/UniqueClientList.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    /**
+     * Returns the backing list as an unmodifiable {@code ObservableList}.
+     */
+    public ObservableList<Client> asUnmodifiableObservableList() {
+        return internalUnmodifiableList;
+    }
+
+    @Override
+    public Iterator<Client> iterator() {
+        return internalList.iterator();
```
Comment (tituswe) [inline src/main/java/seedu/address/model/client/UniqueClientList.java:109]: Kindly add a test case to cover this

File: src/main/java/seedu/address/model/client/UniqueClientList.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof UniqueClientList // instanceof handles nulls
+                        && internalList.equals(((UniqueClientList) other).internalList));
+    }
+
+    @Override
+    public int hashCode() {
+        return internalList.hashCode();
```
Comment (tituswe) [inline src/main/java/seedu/address/model/client/UniqueClientList.java:121]: Kindly add a test case to cover this

File: src/main/java/seedu/address/ui/ClientCard.java [hunk 3]
```diff
@@ -42,17 +42,17 @@ public class PersonCard extends UiPart<Region> {
...
 
     /**
-     * Creates a {@code PersonCode} with the given {@code Person} and index to display.
+     * Creates a {@code ClientCode} with the given {@code Client} and index to display.
      */
-    public PersonCard(Person person, int displayedIndex) {
+    public ClientCard(Client client, int displayedIndex) {
         super(FXML);
-        this.person = person;
+        this.client = client;
```
Comment (tituswe) [inline src/main/java/seedu/address/ui/ClientCard.java:49]: Kindly add a test case to cover this

File: src/main/java/seedu/address/ui/ClientCard.java [hunk 4]
```diff
@@ -42,17 +42,17 @@ public class PersonCard extends UiPart<Region> {
...
-        name.setText(person.getName().fullName);
-        phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
-        email.setText(person.getEmail().value);
-        person.getTags().stream()
+        name.setText(client.getName().fullName);
+        phone.setText(client.getPhone().value);
+        address.setText(client.getAddress().value);
+        email.setText(client.getEmail().value);
+        client.getTags().stream()
```
Comment (tituswe) [inline src/main/java/seedu/address/ui/ClientCard.java:55]: Kindly add a test case to cover this

File: src/main/java/seedu/address/ui/ClientListPanel.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.client.Client;
+
+/**
+ * Panel containing the list of clients.
+ */
+public class ClientListPanel extends UiPart<Region> {
+    private static final String FXML = "ClientListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(ClientListPanel.class);
```
Comment (tituswe) [inline src/main/java/seedu/address/ui/ClientListPanel.java:18]: Kindly add a test case to cover this

File: src/main/java/seedu/address/ui/ClientListPanel.java [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    private ListView<Client> clientListView;
+
+    /**
+     * Creates a {@code ClientListPanel} with the given {@code ObservableList}.
+     */
+    public ClientListPanel(ObservableList<Client> clientList) {
+        super(FXML);
+        clientListView.setItems(clientList);
+        clientListView.setCellFactory(listView -> new ClientListViewCell());
+    }
```
Comment (tituswe) [inline src/main/java/seedu/address/ui/ClientListPanel.java:30]: Kindly add a test case to cover this

File: src/main/java/seedu/address/ui/ClientListPanel.java [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+    /**
+     * Custom {@code ListCell} that displays the graphics of a {@code Client} using a {@code ClientCard}.
+     */
+    class ClientListViewCell extends ListCell<Client> {
```
Comment (tituswe) [inline src/main/java/seedu/address/ui/ClientListPanel.java:35]: Kindly add a test case to cover this

**Review by tituswe — CHANGES_REQUESTED** [review]
Take note of DevGuide and UserGuide. Additional indentation was added, we need to check if it will mess up the CheckStyle

**Review by pangrwa — COMMENTED** [review]
LGTM. Good effort in changing everything from `Person` to `Client`

**Review by tituswe — APPROVED** [review]
Do make sure to update the code cov next, otherwise keep up the good work! LGTM.

**Pull request merged**